opengrok-indexer/src/main/java/org/opengrok/indexer/history/GitRepository.java

```diff
@@ -144,21 +144,23 @@ private HistoryRevResult getHistoryRev(OutputStream out, String fullpath, String
         try (org.eclipse.jgit.lib.Repository repository = getJGitRepository(directory.getAbsolutePath())) {
             ObjectId commitId = repository.resolve(rev);
 
-            // a RevWalk allows to walk over commits based on some filtering that is defined
+            // A RevWalk allows walking over commits based on some filtering that is defined.
             try (RevWalk revWalk = new RevWalk(repository)) {
                 RevCommit commit = revWalk.parseCommit(commitId);
                 // and using commit's tree find the path
                 RevTree tree = commit.getTree();
 
-                // now try to find a specific file
+                // Now try to find a specific file.
                 try (TreeWalk treeWalk = new TreeWalk(repository)) {
                     treeWalk.addTree(tree);
                     treeWalk.setRecursive(true);
                     treeWalk.setFilter(PathFilter.create(filename));
                     if (!treeWalk.next()) {
-                        LOGGER.log(Level.FINEST,
-                                String.format("Did not find expected file '%s' in revision %s for repository '%s'",
-                                        filename, rev, directory));
+                        if (LOGGER.isLoggable(Level.FINEST)) {
```

Comment:
Why not just use:
```java
LOGGER.log(Level.FINEST, "Did not find expected file ''{0}'' in revision {1} for repository ''{2}''", new Object[] {filename, rev, directory}); 
```
In that case the check is hidden and no message formatting is happening. Can be also applied to the warning log below.

Reply:
Thanks !